devscripts/run_tests.sh

```diff
@@ -1,14 +1,4 @@
 #!/usr/bin/env sh
 
-if [ -z "$1" ]; then
-    test_set='test'
-elif [ "$1" = 'core' ]; then
-    test_set="-m not download"
-elif [ "$1" = 'download' ]; then
-    test_set="-m download"
-else
-    echo 'Invalid test type "'"$1"'". Use "core" | "download"'
-    exit 1
-fi
-
-python3 -bb -Werror -m pytest "$test_set"
```

Comment:
Would it be possible to force `-bb -Werror` in run_tests.py?

Reply:
Sure, I made it propagate if you use it when running `run_tests.py` but we can also just pass it everytime if you think thats better

Reply:
pls do